src/popup/router/components/Tooltip.vue

```diff
@@ -0,0 +1,154 @@
+<template>
```

Comment:
Was introducing a `Tooltip` your idea? I think it brings more compexity in testing and developing for now.
If it was your idea, i propose to postpone adding it, there are a lot of edge cases we would need to work through in order to make it perfect.
![image](https://user-images.githubusercontent.com/9008074/177695709-eba0d12c-8c2d-471d-8d01-f074bad7e63d.png)

Reply:
fixed the tooltip, now it will display left, right based on it's position